exploration/semantic-skeletons.md

```diff
@@ -0,0 +1,180 @@
+# Semantic Skeletons Design
+
+Status: **Proposed**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@sffc</dd>
+    <dd>@aphillips</dd>
+		<dt>First proposed</dt>
+		<dd>2024-04-06</dd>
+		<dt>Pull Requests</dt>
+		<dd>#000</dd>
+	</dl>
+</details>
+
+## Objective
+
+_What is this proposal trying to achieve?_
+
+"Semantic skeletons" are a method introduced in CLDR 46 for programmatically selecting a datetime pattern for formatting. 
+There is a fixed set of acceptable semantic skeletons.
+
+Advantages of semantic skeletons over classical skeletons:
+
+- A smaller set of acceptable options focused on producing outputs that are sensical
+- Allows for a more efficient implementation
+- Allows for a more clear, ergonomic API
+- More future-proof since CLDR recently added them (??)
+
+Advantages of semantic skeletons over other mechanisms:
+
+- Unlike "picture strings" (`MMM dd, yyyy`), skeletons do not require translation.
```

Comment:
This is a straw-man argument, as picture strings have not at any point been even suggested for MF2. That should at least be noted somewhere.

Reply:
I can note that. Actually, picture strings were considered in the way-long ago. This text was added to acknowledge their existence (especially in MF1!) and document why we never consider them. There was a discussion of them in yesterday (2025-04-22)'s call which prompted these latest edits. 

I will add a note to point out the WG consensus not to support picture strings.

Reply:
Done